db/data_migration/20150514105208_add_eu_logo_url_to_eafrd_editions.rb

```diff
@@ -0,0 +1,24 @@
+slugs = [
+  "countryside-stewardship-facilitation-funding",
+  "countryside-stewardship-water-capital-grants-2015-capital-items",
+  "countryside-stewardship-water-capital-grants-catchment-sensitive-farming",
+  "countryside-stewardship-woodland-capital-grants-2015",
+  "countryside-productivity-scheme",
+  "rural-development-programme-for-england-leader-funding"
+]
+logo_url = "https://assets.digital.cabinet-office.gov.uk/media/55547b94ed915d15d8000057/european-agricultural-fund-for-rural-development.gif"
```

Comment:
Where does this image assset actually live on the stack? i.e. is it in a repository somewhere? Maybe better it it is included in `app/assets/images` somewhere and then we can specify it as a path and let the asset handling code generate an appropriate URL for it when `image_tag` is called. I think that will be more resilient than a hard-coded asset URL in the database.